src/Tema1-DianaNiculescu.txt

```diff
@@ -0,0 +1,114 @@
+                                                  XPATH LOCATORS
+
+
+1) Locatorul  //button[@onclick='swapCheckbox()'] are urmatoarele:
+- tagname = button
+- attribute = onclick
+- attribute value = swapCheckbox()
+- rol: identifica in pagina "Dynamic Controls"(https://the-internet.herokuapp.com/dynamic_controls) butonul "Remove"
+
+2) Locatorul //div[@id='checkbox'] are urmatoarele:
+- tagname = div
+- attribute = id
+- attribute value = checkbox
+- rol: identifica in pagina "Dynamic Controls"(https://the-internet.herokuapp.com/dynamic_controls) checkbox-ul "A checkbox"
+
+3) Locatorul //input[@class='form-control-file'] are urmatoarele :
```

Comment:
e recomandat sa nu iei clasa ca si atribut, mai ales in acest caz in care avei id ca si atribut. Clasele se pot schimba in timp sau pot aparea mai multe elemente avand aceeasi clasa